influxdb_iox/src/commands/database.rs

```diff
@@ -187,6 +190,23 @@ struct Delete {
     name: String,
 }
 
+/// Disown a database from its current server owner
+#[derive(Debug, StructOpt)]
+struct Disown {
+    /// The name of the database to disown
+    name: String,
+
+    /// Optionally, the UUID of the database to delete. This must match the UUID of the current
+    /// database with the given name, or the disown operation will result in an error.
+    #[structopt(short, long)]
+    uuid: Option<String>,
```

Comment:
And just noticed this can be 

```suggestion
    uuid: Option<Uuid>,
```

and structopt will take care of it... another fix incoming 😅